psi_codex/diagnostics.py

```
@@ -0,0 +1,46 @@
+        "System Status": status
+    }
+
+def generate_priority_report(history):
+    """
+    Generates a formatted text report of system priorities.
+    """
+    priorities = check_priorities(history)
+    report = "\n[bold cyan]=== Ψ-Codex Priority Audit ===[/]\n"
+    for priority, result in priorities.items():
+        color = "green" if "PASS" in result or "COHERENT" in result else "red"
+        if "Data N/A" in result: color = "yellow"
```

Ruff (0.15.6)

[error] Multiple statements on one line (colon)

(E701)